app/src/main/java/com/example/cm/ui/notifications/MeetupNotificationsViewModel.java

```diff
@@ -0,0 +1,59 @@
+package com.example.cm.ui.notifications;
+
+import com.example.cm.data.models.MeetupNotification;
+import com.example.cm.data.models.Notification;
+import com.example.cm.data.repositories.MeetupNotificationRepository;
+import com.example.cm.data.repositories.MeetupRepository;
+
+import java.util.Objects;
+
+public class MeetupNotificationsViewModel extends NotificationsViewModel {
+
+    private final MeetupRepository meetupRepository;
+
+    public MeetupNotificationsViewModel() {
+        super();
+        meetupRepository = new MeetupRepository();
+        notificationRepository = new MeetupNotificationRepository(this);
+        notificationRepository.getNotificationsForUser();
+    }
+
+    @Override
+    public void acceptRequest(Notification notification) {
```

Comment:
why is it the parameter type of notification?
for me the step to create a new MeetupNotification out of the Notification seems a bit strange
I think making seperate repositories makes things more clear